This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Sun, 27 Sep 2020 14:00:00 +0200
Subject: [PATCH 2/2] Correctly annotate data symbols within module-definition

Helps llvm-mingw.

diff --git a/source/x265.def.in b/source/x265.def.in
index 1111111..2222222 100644
--- a/source/x265.def.in
+++ b/source/x265.def.in
@@ -9,9 +9,9 @@ x265_picture_init
 x265_picture_alloc
 x265_picture_free
 x265_param_apply_profile
-x265_max_bit_depth
-x265_version_str
-x265_build_info_str
+x265_max_bit_depth DATA
+x265_version_str DATA
+x265_build_info_str DATA
 x265_encoder_headers
 x265_encoder_parameters
 x265_encoder_reconfig
